src/core_plugins/kibana/public/management/sections/settings/advanced_row.js

```diff
@@ -65,11 +65,11 @@ uiModules.get('apps/management')
         });
       };
 
-      $scope.isValueNonDefault = (conf) => {
-        return !conf.isCustom
-          && conf.value !== undefined
-          && conf.value !== ''
-          && String(conf.value) !== String(conf.defVal);
+      $scope.isDefaultValue = (conf) => {
+        return conf.isCustom
```

Comment:
Yeah, the idea is that if its a custom config setting then there's no notion of a default value. Maybe the `conf.isCustom` check belongs outside this function, in the template instead?